includes/prompt-tags/class-prompt-tags.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Newspack Popups Prompt Tags
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Campaigns;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Prompt Tags class.
+ */
+class Prompt_Tags {
+	/**
+	 * Registered prompt tags.
+	 *
+	 * @var Prompt_Tag[]
+	 */
+	protected static $tags = [];
+
+	/**
+	 * Initialize hooks.
+	 */
+	public static function init_hooks() {
+		add_action( 'init', [ __CLASS__, 'register_default_tags' ] );
+		add_filter( 'newspack_popups_popup_content', [ __CLASS__, 'parse_tags' ] );
+	}
+
+	/**
+	 * Registers default prompt tags.
+	 *
+	 * @return void
+	 */
+	public static function register_default_tags() {
+		self::register_tag(
+			'site_name',
+			[
+				'title'       => __( 'Site Name', 'newspack-popups' ),
+				'description' => __( 'The name of the site.', 'newspack-popups' ),
+				'callback'    => function() {
+					return get_bloginfo( 'name' );
+				},
+			]
+		);
+		self::register_tag(
+			'site_description',
+			[
+				'title'       => __( 'Site Description', 'newspack-popups' ),
+				'description' => __( 'The description of the site.', 'newspack-popups' ),
+				'callback'    => function() {
+					return get_bloginfo( 'description' );
+				},
+			]
+		);
+		self::register_tag(
+			'articles_read',
+			[
+				'title'        => __( 'Articles Read', 'newspack-popups' ),
+				'description'  => __( 'Number of articles read in the last 30 day period.', 'newspack-popups' ),
+				'use_criteria' => true,
```

Comment:
you think it's safe to assume that the tag slug will always match the criteria slug? Or maybe better to declare a string here with the criteria slug you want to use?

Reply:
It should be the case, but to be safe I changed it in 7fd317da4adc1aa354c5c81cd47c2c3e81361ed7